js/google_ima.js

```diff
@@ -1385,6 +1385,16 @@ require("../html5-common/js/utils/utils.js");
        */
       var _onImaAdError = privateMember(function(adError)
       {
+        if (adError &&
+          adError.type === 'adError' &&
+          adError.getErrorCode() === 1009 /*'The VAST response document is empty.'*/ ) {
+          // Empty response happens when 2 or more ads must be loaded simultaneously after playback seek ahead
+          // for 2 or more points of ads. In this case all prev ads loads are cancelled and the only last loaded.
+          // The requests for already cancelled ads seems to return empty VAST. Consider this is not an error
+          // @see https://jira.corp.ooyala.com/browse/PLAYER-3832 discussion
+          return;
```

Comment:
Are we going to add some additional checks in the UI like @jpfranco mentioned in PLAYER-3832? If so, will this part still be needed?